### PR TITLE
Fix BBE related to secured services and clients

### PR DESCRIPTION
**Pull request by ldclakmal**
## Purpose
- Fix BBE of the secured client with basic auth, JWT auth, and oauth2
 - Fix BBE of the secured service with basic auth and JWT auth

### CODE REVIEW

File: examples/secured-client-with-basic-auth/secured_client_with_basic_auth.bal [hunk 1]
```diff
@@ -5,33 +5,30 @@ import ballerina/log;
...
     var response = httpEndpoint->get("/hello/sayHello");
-    match response {
-        http:Response resp => {
-            var result = resp.getPayloadAsString();
-            log:printInfo(result is error ? "Failed to retrieve payload." : result);
-        }
-        error err => log:printError("Failed to call the endpoint.");
+    if (response is http:Response) {
+        var result = response.getPayloadAsString();
+        log:printInfo(result is error ? "Failed to retrieve payload." : result);
```
Comment (VijithaEkanayake) [inline examples/secured-client-with-basic-auth/secured_client_with_basic_auth.bal]:
```suggestion
        log:printInfo((result is error) ? "Failed to retrieve payload." : result);
```

Comment (ldclakmal) [reply examples/secured-client-with-basic-auth/secured_client_with_basic_auth.bal]: Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/11969/commits/cbf3940aa9a9d89f4b77697cabf82405c9c94bf9

File: examples/secured-client-with-basic-auth/secured_client_with_basic_auth.bal [hunk 2]
```diff
@@ -5,33 +5,30 @@ import ballerina/log;
...
-        http:Response resp => {
-            var result = resp.getPayloadAsString();
-            log:printInfo(result is error ? "Failed to retrieve payload." : result);
-        }
-        error err => log:printError("Failed to call the endpoint.");
+    if (response is http:Response) {
+        var result = response.getPayloadAsString();
+        log:printInfo((result is error) ? "Failed to retrieve payload." : result);
+    } else {
+        log:printError("Failed to call the endpoint.");
```
Comment (wggihan) [inline examples/secured-client-with-basic-auth/secured_client_with_basic_auth.bal]: Shall we print the actual error as well that result from get method.

Comment (ldclakmal) [reply examples/secured-client-with-basic-auth/secured_client_with_basic_auth.bal]: Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/11969/commits/ac0fe16ec49c2ab56e5e39177aa86d4e6a352942

**Pull request merged**